client/src/utils/Functions.ts

```diff
@@ -1,6 +1,21 @@
+import { ArtistProp, TrackObjTypes } from '../types';
+
+// helper function to convert ms to mm:ss (song standard)
 export const convertMsToStandardTime = (ms: number) => {
   const minutes = Math.floor(ms / 60000);
   const seconds = ((ms % 60000) / 1000).toFixed(0);
   const addZeroMaybe = Number(seconds) < 10 ? '0' : '';
   return `${minutes}:${addZeroMaybe}${seconds}`;
 };
+
+// helper function to create track obj
+export const createTrackObj = (item: TrackObjTypes) => {
```

Comment:
Probably destructure this one level to simplify the object below a bit... 
```suggestion
export const createTrackObj = ({ id, name, artists, album }: TrackObjTypes) => {
```